packages/runner/src/lib/logger.js

```diff
@@ -2,6 +2,86 @@
 
 import _ from 'lodash'
 
+const _joinLogArgs = (args1, args2) => {
```

Comment:
would be nice to write jsdoc explanation what is going on here with examples. I guess it forms single string in the first place plus arguments to args1 and then args2 but for what...